bisqapps/iosClient/Podfile.lock

```diff
@@ -18,4 +18,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: e4e5bb187d8247572973af90a35b57b84c96b4c6
 
-COCOAPODS: 1.16.2
+COCOAPODS: 1.15.2
```

Comment:
is this intentional? if so, happy to merge it but just would like to understand why the downgrade?

Reply:
Nope!! automated

Reply:
i didnt have up to date cocoapods

Reply:
ok I'll revert that then, let me just test the change without this